tests/spec/fs.stat.spec.js

```diff
@@ -92,3 +92,32 @@ describe('fs.stat', function() {
     });
   });
 });
+
+describe('fs.promises.stat', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fs = util.fs();
+    expect(fs.promises.stat).to.be.a('function');
+  });
+  
+  it('should return a stat object if file exists', function(done){
+    var fs = util.fs();
+
+    fs.promises.stat('/')
+      .then(result => { 
+        expect(result).to.exist;
+        expect(result['node']).to.be.a('string');
+        expect(result['dev']).to.equal(fs.name);
+        expect(result['size']).to.be.a('number');
+        expect(result['nlinks']).to.be.a('number');
+        expect(result['atime']).to.be.a('number');
+        expect(result['mtime']).to.be.a('number');
+        expect(result['ctime']).to.be.a('number');
+        expect(result['type']).to.equal('DIRECTORY');
+        done();
```

Comment:
To make this test more promise-friendly, please remove the unnecessary `done();`.